pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -38,6 +39,11 @@ import (
 	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
 )
 
+var (
+	// defaultCachePeriod is the default cache period for each cpuUsage.
+	defaultCachePeriod = time.Hour
```

Comment:
This seems a little long, since we collect stats for eviction every 10 seconds or so.  Maybe set the cache period to 10 minutes?

Reply:
Yep, 10 min lgtm.